### Finding the Product of 1,230,000 and $$ 0.4 \times 10^{-8} $$  

The first thing we need to do is we need to convert the first number into scientific notation. 

We can do this by simply saying:

$$ 1,230,000 \rightarrow 1.23 \times 10^{6} $$ 

Then we put the numbers side by side 

$$ (1.23 \times 10^{6} ) * (0.4 \times 10^{-8} ) $$ 

$$ (1.23 * 0.4) \times (10^{6 - 8}) $$ 

In [1]:
1.23 * 0.4

0.492

We need to change this first product into scientific notation. 

$$ 0.492 \rightarrow 4.92 \times 10^{-1} $$ 

To get the final answer, we just have to account for the $$ 10^{-2} $$  that comes from the base 10s of the product. 

$$ 4.92 \times 10^{-3} $$ 